generator/internal/discovery_resource.cc

```diff
@@ -138,21 +138,31 @@ StatusOr<std::string> DiscoveryResource::FormatRpcOptions(
   }
   rpc_options.push_back(absl::StrCat(http_option, "    };"));
 
-  std::vector<std::string> params =
+  std::vector<std::string> parameter_order =
       method_json.value("parameterOrder", std::vector<std::string>{});
-  if (!params.empty()) {
+  if (!parameter_order.empty()) {
+    // Workaround for necessary, but not marked REQUIRED, mask field for update
```

Comment:
Is there an AIP or some bug we can reference in the future?  The change makes sense, just trying to avoid "what in the world" moments for our future selves.